meson_options.txt

```diff
@@ -2,3 +2,4 @@ option('tests', type : 'boolean', value : true, description : 'Build self tests'
 option('vapi', type : 'boolean', value : true, description : 'Build VAPI')
 option('usb_ids', type : 'string', value : '/usr/share/hwdata/usb.ids', description : 'Path to usb.ids file')
 option('docs', type : 'boolean', value : true, description : 'Generate documentation')
+option('gir', type : 'boolean', value : true, description : 'Generate gobject introspection data')
```

Comment:
I think other projects just use `introspection` rather than `gir`...

Reply:
Checking the packages we have at Void it seems to be true:

```
$ rg build_style=meson -l | xargs rg -l -- '-Dintrospection'
srcpkgs/appstream-glib/template
srcpkgs/atk/template
srcpkgs/eog/template
srcpkgs/fwupd/template
srcpkgs/gcab/template
srcpkgs/gnome-bluetooth/template
srcpkgs/gnome-todo/template
srcpkgs/gssdp/template
srcpkgs/json-glib/template
srcpkgs/libcloudproviders/template
srcpkgs/libgepub/template
srcpkgs/libgit2-glib/template
srcpkgs/libhandy/template
srcpkgs/libnice/template
srcpkgs/libxmlb/template
srcpkgs/nautilus/template
srcpkgs/playerctl/template
srcpkgs/spice-gtk/template
$ rg build_style=meson -l | xargs rg -l -- '-Dgir'
srcpkgs/AppStream/template
srcpkgs/gdk-pixbuf/template
```

will adjust accordingly.